glib/src/object.rs

```diff
@@ -1106,6 +1106,10 @@ glib_object_wrapper!(@object
 pub type ObjectClass = Class<Object>;
 
 impl Object {
+    /// Create a new instance of an object with the given properties.
+    ///
+    /// This fails if the object is not instantiable, doesn't have all the given properties or
+    /// property values of the wrong type are provided.
```

Comment:
```suggestion
    /// Create a new instance of an object with the given properties.
    ///
    /// # Panics
    ///
    /// This fails if the object is not instantiable, doesn't have all the given properties or
    /// property values of the wrong type are provided.
```

I think something like this would look nicer & make it clear that it's the cause of a potential panic

Reply:
Indeed, I missed that this one doesn't return a `Result` but just panics :)

Reply:
Hm no, this returns a `Result`. It won't panic by itself.